db/sql/migrations/updates/20181026095901_change_wsport_type_in_peers.sql

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+
+/*
+  DESCRIPTION: ALTER TABLE peers ALTER COLUMN "wsPort" TYPE INT;
```

Comment:
Can you update the description with something more informative than same DML statement? ex: `Port numbers are 16 bit, which means we have a theoretical limit of 65536`

Reply:
Agree we should explain why or how this migration will impact the system

Reply:
Updated. Check please.